packages/ui/src/component/UICanvas.ts

```
@@ -306,7 +314,13 @@
         this._sortDistance = this._distance;
         break;
       case CanvasRenderMode.WorldSpace:
-        this._sortDistance = Vector3.distance(cameraPosition, this.entity.transform.worldPosition);
+        const boundsCenter = this._getBounds().getCenter(UICanvas._tempVec3);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)